agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3678,23 +3679,22 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             break;
         }
 
-        auto apconfHeader =
-            cmdu_tx.create(0, ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WSC_MESSAGE);
-        auto tlvAp = cmdu_tx.addClass<wfa_map::tlvApRadioBasicCapabilities>();
-        if (tlvAp == nullptr) {
+        auto c  = cmdu_tx.create(0, ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WSC_MESSAGE);
+        auto ap = cmdu_tx.addClass<wfa_map::tlvApRadioBasicCapabilities>();
+        if (ap == nullptr) {
             LOG(ERROR) << "Error creating TLV_AP_RADIO_BASIC_CAPABILITIES";
             return false;
         }
         std::copy_n(network_utils::mac_from_string(config.radio_identifier).oct, sizeof(sMacAddr),
```

Comment:
This line needs to be changed now to:
ap->radio_uid() = network_utils::mac_from_string(config.radio_identifier);

Reply:
Nice one, thanks. done.